examples/ads.amp.html

```diff
@@ -906,6 +907,18 @@ <h2 id="yieldmo">Yieldmo</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="webediads">Webediads (private ad network)</h2>
```

Comment:
I saw this example is an ad with `no-content`. Could you fix it? we need a rendered ad to test here. Thanks.